pkg/openstack-cinder/config/cloudinfo.go

```
@@ -87,10 +85,12 @@ func getCloudInfo() (*CloudInfo, error) {
 	// our assets mount the CA file at a known path and we don't want to rely on other things
 	// setting the 'cafile' value in our clouds.yaml file to the same path, so we explicitly
 	// override this if a CA file is present
+	// NOTE(stephenfin): gophercloud (or rather, the clientconfig package) doesn't currently
+	// provide the way to override configuration other than via environment variables
 	if _, err := os.Stat(caFile); err == nil {
-		// NOTE(stephenfin): gophercloud (or rather, the clientconfig package) doesn't currently
-		// provide the way to override configuration other than via environment variables
 		os.Setenv("OS_CACERT", caFile)
```

golangci-lint (2.12.1)

[error] Error return value of `os.Setenv` is not checked

(errcheck)